Ojete.html

```diff
@@ -3,13 +3,14 @@
      <head>
        <meta charset = "utf-8"/>
        <title>Jonás is God</title>
+	<link rel="stylesheet" href="estilo.css">
      </head>
      <body>
        <h1>Cuñaoooos Programmer</h1>
 	<img src="Jonas_God.jpeg" width="100" height="100">
        <p>Me encanta github porque:</p>
        <ol>
-	<li>Mola mucho</li>
+	<li class= "azul">Mola mucho</li>
         <li>Soy Freak</li>
         <li>Mi cuñao es más freak</li>

```

Comment:
Te habia dicho que la SEGUNDA linea fuera azul, no la primera!

```suggestion
	<li>Mola mucho</li>
        <li class= "azul">Soy Freak</li>
        <li>Mi cuñao es más freak</li>

```